EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,82 @@
+eip: xxxx  
+title: Transaction Gas Limit Cap  
+description: Introduce a protocol-level cap on the maximum gas used by a transaction to 30 million.  
+author: Giulio Rebuffo (@Giulio2002)  
+discussions-to: [Insert discussion forum link here]  
```

Comment:
Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5